.github/workflows/check-packetbeat.yml

```diff
@@ -0,0 +1,30 @@
+name: check-packetbeat
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/check-packetbeat.yml'
+      - 'packetbeat/**'
+      - 'x-pack/packetbeat/**'
+
+env:
+  BEAT_MODULE: 'packetbeat'
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Fetch Go version from .go-version
+      run: echo "GO_VERSION=$(cat .go-version)" >> $GITHUB_ENV
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ${{ env.GO_VERSION }}
+    - name: Install libpcap-dev
+      run: sudo apt-get install -y libpcap-dev
+    - name: Run check/update
+      run: |
+        export PATH=$PATH:$(go env GOPATH)/bin
+        go install github.com/magefile/mage@latest
+        make -C ${{ env.BEAT_MODULE }} check update
+        make check-no-changes
```

Comment:
Same question here, it's either was incorrect before or it's incorrect now, before the list of checks was:

```sh
# what you also have here
make -C packetbeat check update;
make check-no-changes;
# what was additionally in the Jenkins file
cd x-pack/packetbeat;
mage check;
mage update;
```

Perhaps this checks below are included into those make targets that you use now. In this case would be nice to have it consistent with `.github/workflows/check-xpack-packetbeat.yml`, now they are different and I don't expect them to be any different.

Reply:
See my previous comment https://github.com/elastic/beats/pull/32711#discussion_r947800792